test/index.js

```diff
@@ -1,8 +1,7 @@
-"use strict";
+import assert from 'power-assert';
 
-describe('circleCiのテスト', function() {
-	it('circleCiが動作している', function(done) {
-		(5).should.be.exactly(5);
-		done();
-	});
+describe('mochaのテスト', ()=> {
+  it('テストが動作している', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).